frontend/packages/metal3-plugin/src/components/dashboard/inventory-card.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import { DashboardItemProps } from '@console/internal/components/dashboards-page/with-dashboard-resources';
+import { DashboardCard } from '@console/internal/components/dashboard/dashboard-card';
+import { DashboardCardBody } from '@console/internal/components/dashboard/dashboard-card/card-body';
+import { DashboardCardHeader } from '@console/internal/components/dashboard/dashboard-card/card-header';
+import { DashboardCardTitle } from '@console/internal/components/dashboard/dashboard-card/card-title';
+import {
+  InventoryItem,
+  ResourceInventoryItem,
+} from '@console/internal/components/dashboard/inventory-card/inventory-item';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { FirehoseResource } from '@console/internal/components/utils';
+
+import { PodModel } from '@console/internal/models';
+import { getPodStatusGroups } from '@console/internal/components/dashboard/inventory-card/utils';
+import { getHostStorage, getHostNICs, getHostCPU } from '../../selectors';
+
+const k8sResources: FirehoseResource[] = [
+  {
+    isList: true,
+    kind: PodModel.kind,
+    prop: 'pods',
+  },
+];
+
+export const InventoryCard: React.FC<InventoryCardProps> = ({
+  obj,
+  watchK8sResource,
+  stopWatchK8sResource,
+  resources,
+}) => {
+  React.useEffect(() => {
+    k8sResources.forEach((r) => watchK8sResource(r));
+    return () => {
+      k8sResources.forEach((r) => stopWatchK8sResource(r));
+    };
+  }, [watchK8sResource, stopWatchK8sResource]);
+
+  const pods = _.get(resources, 'pods');
+  const podsLoaded = _.get(pods, 'loaded');
+  const podsData = _.get(pods, 'data', []) as K8sResourceKind[];
+
+  const diskCount = getHostStorage(obj).length;
+  const nicCount = getHostNICs(obj).length;
+  const cpuCount = getHostCPU(obj).count;
```

Comment:
this does not seem correct

Reply:
Please elaborate. What do you expect it to be?

Reply:
Ah, thats an object. Sorry :)